src/components/Checkout/DonationItem.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+	<div class="basket-donation-item row">
+		<span class="small-3 columns">
+			<span class="donation-icon">
+				<kv-icon class="dedicate-heart" name="dedicate-heart" />
+			</span>
+		</span>
+		<span class="small-9 medium-6 columns">
+			<div class="donation-info featured-text">
+				Donation to Kiva
+			</div>
+			<div>
+				<a
+					class="small-text donation-help-text"
+					@click.prevent="triggerDefaultLightbox">
+					Help Kiva reach more borrowers around the world
+				</a>
+				<!-- This lightbox will be replaced with a Popper tip message. -->
+				<kv-lightbox
+					:visible="defaultLbVisible"
+					@lightbox-closed="lightboxClosed">
+					<h2 slot="title">How does Kiva use donations?</h2>
+					<div>
+						100% of every dollar you lend on Kiva goes directly to funding loans.
+						We rely on small optional donations from you and others to keep Kiva running.
+						Every $1 donated to Kiva makes $8 in loans possible around the world.
+						Your donation will enable us to:
+						<ul class="disc-list">
+							<li>Send expert staff to over 60 countries to vet and monitor loans and partners.</li>
+							<li>Build and maintain a website that facilitates over $1 million in loans each week.</li>
+							<li>Provide comprehensive customer support to thousands of lenders worldwide.</li>
+							<li>Train and support hundreds of volunteers -- 4 for every 1 staff member at Kiva.</li>
+						</ul>
+						If you live in the United States, your donation is tax-deductible.
+						Thank you for considering a donation to Kiva.
+					</div>
+				</kv-lightbox>
+			</div>
+		</span>
+		<span class="small-12 medium-4 columns featured-text">
+			<div class="small-12 medium-6 align-center">$
+				<input
+					type="input"
+					class="donation-amount-input"
+					name="donation"
+					id="donation"
+					v-model="amount">
```

Comment:
prob should add some validation at to this input at some point...need to determine what that would be

Reply:
I just created feck-184 to cover the validation rules for this input field.